### ISSUE TITLE
cant load build from sarna

### ISSUE THREAD

**Issue by AweDeus** [fat4eyes-mwo/mwo-simulator#20]
can't load build like this http://mwo.smurfy-net.de/mechlab#i=493&l=2e06bfb18195ee621ad93a520c957268b7e2d9d3

i think it's cuz Single sinks.

**Comment by fat4eyes-mwo** [comment]
Confirmed, could be caused by missing clan single heat sink in module file. Investigating.

**Comment by fat4eyes-mwo** [comment]
Yeah it was single heat sinks that was causing it. Should be handled in https://github.com/fat4eyes-mwo/mwo-simulator/commit/6121e0568762745bd915c47776421bf21c899423

The 'fix' is actually just a workaround, it uses IS single heat sink data for clan single heat sinks. Fortunately I believe clan and IS single heat sinks have identical stats. Unfortunately this could change in the future. I'm keeping this issue open until I confirm with smurfy that IS and Clan single heat sinks are identical.
